core/services/src/main/java/org/exoplatform/services/cms/documents/job/TrashCleanerJob.java

```diff
@@ -0,0 +1,226 @@
+package org.exoplatform.services.cms.documents.job;
```

Comment:
I didn't mention this in the maintenance PR, shouldn't we have a header for this added file  ?

Reply:
Yes we should. I add it